README.md

```diff
@@ -8,6 +8,19 @@ This repo contains:
 * Instructions on how to install Nx Cloud using Helm
 * Instructions on how to install Nx Cloud using kubectl. See [here](./no-helm/README.md).
 
+## An important note on updates!
+
+Please note, as of version `0.11.0` this helm chart is no longer compatible with legacy frontend containers by default. 
+
+**To avoid issues, please ensure the container version you are targeting is >=** `2308.22.7`.
+
+### Compatabiltiy Matrix
+
+| Chart Version | Compatible Images                  |
+|---------------|------------------------------------|
+| <= `0.10.11`  | `2306.01.2.patch4` **and earlier** |
+| >= `0.11.0`   | `2308.22.7` **and later**          |
+
```

Comment:
should we also add a [NOTES.txt](https://helm.sh/docs/chart_template_guide/notes_files/) file in this new version to **explicitly** print out in their terminal that they're running a new Helm chart version and they need Docker version `2308.22.7` and above